conda_forge_tick/update_upstream_versions.py

```diff
@@ -23,6 +24,23 @@
 logger = logging.getLogger("conda-forge-tick._update_versions")
 
 
+def version_match(version, new_version: str, payload):
```

Comment:
This only seems to need the version pattern from the bot, so I would pass that in here.

Reply:
Yup you are right, the reason I oppened the manager here was to perform a count on the number of version performed (like `version_1 - version_2 >= N` or something)